### DISCUSSION TITLE
trying to connect to server

### DISCUSSION THREAD

**Discussion by serifas** [Burralis/Game-Launcher#50] [General]
hello I am trying to see if I can get some help figuring out how to connect to my server.
I setup a cert with certbot, arctium says the url is okay but when I get in game it won't connect me to the server still.
I have everything setup on my database as far as the realm goes.
I also can connect locally with the same core,
Is there a way to connect remotely?
I am using the launcher posted here
https://github.com/Arctium/WoW-Launcher/discussions/48

**Comment by Fabi** [comment]
NO support for your server config issues here.
